lib/specinfra/command/sles/v12/service.rb

```diff
@@ -2,4 +2,8 @@ class Specinfra::Command::Sles::V12::Service < Specinfra::Command::Sles::Base::S
   class << self
     include Specinfra::Command::Module::Systemd
   end
+
+  def check_is_enabled(service, level=3)
+    "systemctl list-unit-files | grep #{escape(service)}\\.service | grep enabled"
+  end
 end
```

Comment:
You don't need to define this method, bacuse this class includes Specinfra::Command::Module::Systemd.